addons/mail/static/src/core/web/message_patch.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
     <t t-inherit="mail.Message" t-inherit-mode="extension">
-        <xpath expr="//div[hasclass('o-mail-Message-textContent')]/*[1]" position="before">
+        <xpath expr="//t[@name='notification body']" position="replace">
```

Comment:
`notification body` => `bodyAsNotification`

So that it matches exactly the `t-name` without the patch